bedrock/careers/templates/careers/includes/testimonials.html

```diff
@@ -5,50 +5,52 @@
 #}
 {% block careers_content %}
   <section class="c-careers-testimonials c-careers-section no-js-hidden">
-    <div class="mzp-l-content">
-      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two c-careers-testimonials-wrapper">
-        <li class="has-modal" data-modal-id="perla-d">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+      <ul class="mzp-l-content c-careers-testimonials-wrapper">
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="perla-d">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
+          </q>
           <a href="#perla-d-testimonial">
-          Read full testimonial
+          Read full Quote
           </a>
         </li>
-        <li class="has-modal" data-modal-id="lauren-n">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+        <li class="c-careers-testimonial-image"></li>
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="lauren-n">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
+          </q>
           <a href="#lauren-n-testimonial">
-          Read full testimonial
+          Read full Quote
           </a>
         </li>
-        <li class="has-modal" data-modal-id="chance-y">
-          <p>
-            "Mozilla are 100% in support of remote work and deeply values remote workers like me... "
-          </p>
+        <li class="c-careers-testimonial-image"></li>
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="chance-y">
+          <q>
+            Mozilla are 100% in support of remote work and deeply values remote workers like me...
+          </q>
           <a href="#chance-y-testimonial">
-          Read full testimonial
+          Read full Quote
           </a>
         </li>
-        <li class="has-modal" data-modal-id="anna-y">
-          <p>
-            "My experience with Mozilla has been only empowering and uplifting one... "
-          </p>
+        <li class="c-careers-testimonial-image"></li>
```

Comment:
**Suggestion (design):** Elise and I discussed and we agreed that it would be better to remove the images from mobile, since there's already a lot of content to sort through and the images would just add bloat

Reply:
Also important to note that since colors are placed based on grid placement, let's make sure that there are colors for each quote in mobile instead of a white background

Reply:
Yup its already configured that way